util/markov.py

```diff
@@ -0,0 +1,69 @@
+from markovchain import JsonStorage
+from markovchain.text import MarkovText, ReplyMode
+import json
+
+import markovify
+
+
+def markovchain_example():
+    markov = MarkovText()
+
+    with open('word_generation/definitions.txt') as fp:
+        for line in fp:
+            markov.data(line, part=True)
+    markov.data('', part=False)
+
+    print(markov(max_length=16) + '\n')
+    print(markov(max_length=16, reply_to='sentence start',
+                 reply_mode=ReplyMode.END) + '\n')
+
+
+def markovify_example():
+    with open('word_generation/definitions.txt') as f:
+        text = f.read()
+
+    neighbor_model = markovify.Text(text)
+
+    print(neighbor_model.make_sentence() + '\n')
+    print(neighbor_model.make_short_sentence(280) + '\n')
+
+
+def make_model():
+    with open('word_generation/dictionary_model.json') as f:
+        dictionary_model_json = json.load(f)
+        dictionary_model = markovify.Text.from_json(dictionary_model_json)
+
+    with open('word_generation/definitions.txt') as f:
+        definitions = f.read()
+    neighbor_model = markovify.Text(definitions, well_formed=False)
+
+    # Combined model weighted 1 to 3 towards the neighbor_model to be more like the neighbor definitions
+    combined_model = markovify.combine(
+        [dictionary_model, neighbor_model], [1, 3])
+    for _ in range(5):
+        print(combined_model.make_sentence() + '\n')
+
+
+def convert_json_to_txt():
```

Comment:
This function and the one below it are only for creating the dictionary_model.json file. Once we figure out if we need to do any text cleanup or make any changes, we could probably get rid of these, since the dictionary model won't change.